todo_app/test/test_db.py

```diff
@@ -1,14 +1,22 @@
 from dotenv import load_dotenv, find_dotenv
 from todo_app import app
 import mongomock
-
+import pytest
 
 @pytest.fixture
 def client():
     file_path = find_dotenv('.env.test')
     load_dotenv(file_path, override=True)
     
-    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
+    with mongomock.patch(servers=(('mongodb+srv://fakemongo.com', 27017),)):
```

Comment:
I believe you just want the server name here:
```suggestion
    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
```

Reply:
Thanks jack - have done this :)